.clang-tidy

```diff
@@ -0,0 +1,37 @@
+Checks: >
+    '-*,
+     bugprone-*,
+     clang-analyzer-*,
+     clang-diagnostic-*,
+     cppcoreguidelines-*,
+       -cppcoreguidelines-avoid-c-arrays,
+       -cppcoreguidelines-avoid-goto,
+       -cppcoreguidelines-avoid-magic-numbers,
+       -cppcoreguidelines-avoid-non-const-global-variables,
+       -cppcoreguidelines-init-variables,
+       -cppcoreguidelines-interfaces-global-init,
+       -cppcoreguidelines-macro-usage,
+       -cppcoreguidelines-no-malloc,
+       -cppcoreguidelines-non-private-member-variables-in-classes,
+       -cppcoreguidelines-owning-memory,
+       -cppcoreguidelines-pro-*,
+     modernize-*,
+       -modernize-avoid-c-arrays,
+       -modernize-use-trailing-return-type,
+       -modernize-use-using,
+     performance-*,
+     readability-*,
+       -readability-braces-around-statements,
+       -readability-else-after-return,
+       -readability-function-cognitive-complexity,
+       -readability-function-size,
+       -readability-identifier-length,
+       -readability-implicit-bool-conversion,
+       -readability-isolate-declaration,
+       -readability-magic-numbers,
+       -readability-named-parameter,
+       -readability-simplify-boolean-expr,
+     mpi-*,
+     openmp-*'
+
+HeaderFilterRegex: '[^n][^o][^l][^i][^n][^t]\.H$'
```

Comment:
Do note that the `-header-filter` CLI arg overrides this value when run from CI (not sure if that's intentional)

Reply:
no, it wasn't intentional.  I just copied the AMReX config

Reply:
removed